.github/workflows/_docs.yml

```diff
@@ -143,3 +146,13 @@ jobs:
           s3-bucket: doc-previews
           path: cppdocs/
           s3-prefix: pytorch/${{ github.event.pull_request.number }}/cppdocs
+
+      - name: Upload functorch Docs Preview
+        uses: seemethere/upload-artifact-s3@v5
+        if: ${{ github.event_name == 'pull_request' && matrix.docs_type == 'functorch' && steps.build-docs.outcome == 'success' }}
+        with:
+          retention-days: 14
+          s3-bucket: doc-previews
+          if-no-files-found: error
+          path: functorch_ghpages/nightly/
+          s3-prefix: pytorch/${{ github.event.pull_request.number }}/functorchdocs
```

Comment:
Appears to work: https://docs-preview.pytorch.org/85838/functorchdocs/index.html

How would I go about adding a link to 
<img width="934" alt="image" src="https://user-images.githubusercontent.com/5652049/192898664-a0f289e1-6bda-4865-b734-3603ae8b1407.png">

?